dali/operators/image/crop/crop_mirror_normalize.cc

```diff
@@ -38,16 +39,24 @@ normalization only.
   .DeprecateArg("image_type", true)  // deprecated since 0.24dev
   .DeprecateArgInFavorOf("output_dtype", "dtype")  // deprecated since 0.24dev
   .AddOptionalArg("dtype",
-    R"code(Output data type. Supported types: `FLOAT` and `FLOAT16`)code", DALI_FLOAT)
+       R"code(Output data type.
+
+By default, the same data type as the input will be used.
+Here are the supported types:
+- `FLOAT`
+- `FLOAT16`
+- `UINT8`)code", DALI_FLOAT)
   .AddOptionalArg("output_layout",
-    R"code(Output tensor data layout)code", TensorLayout("CHW"))
+    R"code(Tensor data layout for the output.)code", TensorLayout("CHW"))
   .AddOptionalArg("pad_output",
-    R"code(Whether to pad the output to number of channels being a power of 2.)code", false)
+    R"code(Determines whether to pad the output to the number of channels as a power of 2).)code", false)
   .AddOptionalArg("mirror",
-    R"code(Mask for horizontal flip.
-- `0` - do not perform horizontal flip for this image
-- `1` - perform horizontal flip for this image.
-)code",
+    R"code(Mask for the horizontal flip.
+
+Here are the values:
+
+- `0`:  Do not perform a horizontal flip for this image.
+- `1`: perform a horizontal flip for this image.)code",
```

Comment:
That's way too verbose and inaccurate (any nonzero value will trigger flipping)
```suggestion
    R"code(If nonzero the image will be flipped (mirrored) horizontally.)code",
```

Reply:
Done